src/ExposureHistory/History/index.tsx

```diff
@@ -56,6 +62,25 @@ const History: FunctionComponent<HistoryProps> = ({
     navigation.navigate(ExposureHistoryStackScreens.MoreInfo)
   }
 
+  const showEnableExposureNotificationsAlert = () => {
+    Alert.alert(
+      t("exposure_notification_alerts.cant_check_for_exposures_title"),
+      t("exposure_notification_alerts.cant_check_for_exposures_body"),
+      [
+        {
+          text: t("common.back"),
+          style: "cancel",
+        },
+        {
+          text: t(
+            "exposure_notification_alerts.cant_check_for_exposures_button",
```

Comment:
what do you think about, `Enable Exposure Notifications` here? in the case that the request succeeds, 'Tell me how' would end up not being very clear as no new dialog would pop up.

Reply:
How could we be in a state where the request succeeds? I think the only other think that could happen is the native enable ENs alert pops up.